src/interpolation.jl

```diff
@@ -1620,32 +1651,58 @@ function lagrange_space(domain,order;
                   dirichlet_boundary,
                   space,
                   major,
-                  shape)
+                  shape,
+                  cache) |> setup_space
+
+end
+
+function setup_space(space::AbstractSpace)
+    if space.cache !== nothing
+        return space
+    end
+    face_dofs = GT.face_dofs(space)
+    cache = space_cache(;face_dofs)
+    replace_cache(space,cache)
+end
 
+function replace_cache(space::AbstractSpace,cache)
```

Comment:
Hi @Cmurilochem This should be only for LagrangeSpace

Reply:
Thanks @fverdugo for pointing this out; that is true indeed! Fixed!